packages/back-end/src/models/GithubIntegration.ts

```diff
@@ -0,0 +1,96 @@
+import { omit } from "lodash";
+import uniqid from "uniqid";
+import mongoose from "mongoose";
+import {
+  GithubIntegrationInterface,
+  CreateGithubIntegrationInput,
+} from "../../types/github";
+import { OrganizationInterface } from "../../types/organization";
+import { fetchRepositories } from "../services/github";
+import { doesTokenExist } from "./GithubUserTokenModel";
+
+type GithubIntegrationDocument = mongoose.Document & GithubIntegrationInterface;
+
+const githubIntegrationSchema = new mongoose.Schema({
+  id: String,
+  organization: String,
```

Comment:
Should this be project scoped?  Thinking about an org with 2 projects - `app1` and `app2`.  If they are changing a feature in `app1`, there's no need to scan the GitHub repo for `app2`

Reply:
Conceptually I don't think an **integration** should be project scoped - an org will have only one integration (its only set up once). When we start to implement the code references part of this feature, I think we'd probably introduce a new data model such as `GithubRepo` that can be project scoped and handle storing data related to code references